plugins/techdocs-backend/src/techdocs/stages/generate/helpers.ts

```diff
@@ -47,6 +47,14 @@ export async function runDockerContainer({
   dockerClient,
   createOptions,
 }: RunDockerContainerOptions) {
+  try {
+    await dockerClient.ping();
+  } catch (e) {
+    throw new Error(
+      `This operation requires Docker. Docker does not appear to be available. Docker.ping() failed with: ${e.message}`,
+    );
+  }
+
```

Comment:
Where does this error show up for the user?

Reply:
If the user tries to generate docs and docker isn't installed an error message is shown rather than the docs. Like this user reports: https://discordapp.com/channels/687207715902193673/714754240933003266/755766269051011162